ui/apps/platform/src/services/ExceptionConfigService.ts

```diff
@@ -2,7 +2,7 @@ import axios from './instance';
 
 const vulnBaseUrl = '/v1/config/private/deferral/vulnerabilities';
```

Comment:
@md2119 If the API URL for this feature is going to change as well we can coordinate on a follow up PR to keep the API and UI in sync.

Reply:
Yes, I plan to change the API. Would it be complete to grep `private/deferral` to find all the occurrences of the API?

Reply:
For the UI yes, at present time there are only two files that contain that string, a service file and a test file.